CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 ### Changes
 - [#1126](https://github.com/LayerManager/layman/issues/1126) Endpoint [GET Workspace Map Thumbnail](https://github.com/LayerManager/layman/blob/v2.1.0/doc/rest.md#get-workspace-map-thumbnail) was removed and replaced with endpoint [GET Map Thumbnail](doc/rest.md#get-map-thumbnail) endpoint to use UUID-based URL `/rest/maps/{uuid}/thumbnail` instead of workspace&name-based URL.
 - [#1126](https://github.com/LayerManager/layman/issues/1126) Endpoint [GET Workspace Layer Thumbnail](https://github.com/LayerManager/layman/blob/v2.1.0/doc/rest.md#get-workspace-layer-thumbnail) was removed and replaced with endpoint [GET Layer Thumbnail](doc/rest.md#get-layer-thumbnail) endpoint to use UUID-based URL `/rest/layers/{uuid}/thumbnail` instead of workspace&name-based URL.
+- [#1126](https://github.com/LayerManager/layman/issues/1126) Endpoint [GET Workspace Map File](https://github.com/LayerManager/layman/blob/v2.1.0/doc/rest.md#get-workspace-map-file) was removed and replaced with endpoint [GET Map File](doc/rest.md#get-map-file) endpoint to use UUID-based URL `/rest/map/{uuid}/file` instead of workspace&name-based URL.
```

Comment:
```suggestion
- [#1126](https://github.com/LayerManager/layman/issues/1126) Endpoint [GET Workspace Map File](https://github.com/LayerManager/layman/blob/v2.1.0/doc/rest.md#get-workspace-map-file) was removed and replaced with endpoint [GET Map File](doc/rest.md#get-map-file) to use UUID-based URL `/rest/map/{uuid}/file` instead of workspace&name-based URL.
```